### ISSUE TITLE
Help please

### ISSUE THREAD

**Issue by duderz2011** [jkaberg/tvhProxy#49]
can some one please explain something i have downloaded the tvhproxy-master ive adjusted the .py file when i follow instructions for setup its not finding the files where am i supposed to put them directory wise. as you can tell im a full on newbie

**Comment by jwrober** [comment]
The readme does leave a few details out. By default the python script intends to run as the `tvh` user in `/home/tvh/tvhProxy` directory.

Install python deps first

```
sudo apt-get install -y git python python-pip python-gevent python-flask python-requests python-virtualenv
```

Create the user

`sudo useradd -m tvh -s /bin/bash`

become the user

`sudo su - tvh`

get the files and set up the environment

```
git clone https://github.com/jkaberg/tvhProxy.git
cd tvhProxy
virtualenv venv
. venv/bin/activate
pip install setuptools --upgrade
pip install wheel
pip install -r requirements.txt
```

At this point you will need to edit the `tvhProxy.py` script and change the `tvhURL` line to match the user id and password for your tvheadend install.

Then you can follow the readme instructions to install the service file to run as a daemon.

**Comment by duderz2011** [comment]
thank you for your reply i have followed your instructions where do i find the py file to adjust settings

**Comment by jwrober** [comment]
its in the tvhProxy directory

**Comment by duderz2011** [comment]
Yes but how do I access this? I downloaded the tvhproxy-master but Ubuntu downloads it itself and installs so how can I access it in Ubuntu to adjust the settings

**Comment by duderz2011** [comment]
I know this is probably so simple but this is all so new to me

**Comment by geordieeddie** [comment]
hey so this is probably something ive managed to mess up, but what is the password for user tvh? ive tried my root password and also tried my password for tvheadend.
tia

**Comment by duderz2011** [comment]
how did you get it installed its not finding the .requirements

**Comment by duderz2011** [comment]
login for tvh is admin and admin

**Comment by roryKell** [comment]
hey geordieeddie - i wonder is your user/pass problem something do with creating the user tvh. 

When creating your ubuntu setup, instead of admiistrator or *youname* as the main account use "tvh". Now when your following the guide above use that account and you should have no problem with the password.
